twisted/twisted
# #11567 Add templates for new issues.

## Scope

Fixes #11567

This is the initial suggestion for some templates.

They are simple text templates.

## Changes

The following templates were created with the following tags auto applied:

* bug  - `bug`
* task - NO-LABEL
* release blocker - `bug` and `release-blocker`
* documentation  (for any typos, comments, docs improvements) - `documentation`


Web forms are not yet used - [release-blocker](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)




## How to test

There is no way for end to end testing until this is merged,

So maybe make sure this looks ok in general, merge it and then after a hands-on testing, we can update what is left in a separate PR.

With these changes, when a new report is made, users must select one of the available options.
The option of a blank issue is disabled.
